tests/python/gaia-ui-tests/gaiatest/apps/clock/regions/alarm.py

```diff
@@ -2,77 +2,63 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import time
-
 from marionette_driver import expected, By, Wait
 
 from gaiatest.apps.clock.app import Clock
+from gaiatest.apps.base import PageRegion
 
 
-class NewAlarm(Clock):
-
-    _alarm_name_locator = (By.ID, 'alarm-name')
-    _repeat_menu_locator = (By.ID, 'repeat-menu')
-    _sound_menu_locator = (By.ID, 'sound-menu')
-    _snooze_menu_locator = (By.ID, 'snooze-menu')
-    _done_locator = (By.ID, 'alarm-done')
-    _time_button_locator = (By.XPATH, "//li[input[@id='time-select']]")
+class Alarm(Clock):
```

Comment:
It makes sense but I wanted to use the functionality of the Clock __init__. Also, when I thought about the actual views, they seem to be views derived from the clock view rather than just a region of the Clock view. They are an entire view for themselves. That's why I stuck with them inheriting from Clock. What say you @JohanLorenzo?